.dassie/app/indexers/generic_work_resource_indexer.rb

```diff
@@ -3,8 +3,9 @@
 # Generated via
 #  `rails generate hyrax:work_resource GenericWorkResource`
 class GenericWorkResourceIndexer < Hyrax::ValkyrieWorkIndexer
-  include Hyrax::Indexer(:basic_metadata)
-  include Hyrax::Indexer(:generic_work_resource)
+  include Hyrax::Indexer(:basic_metadata) unless Hyrax.config.flexible?
+  include Hyrax::Indexer(:generic_work_resource) unless Hyrax.config.flexible?
+  include Hyrax::Indexer('GenericWorkResource') if Hyrax.config.flexible?
```

Comment:
I would naively expect these lines to do the same thing, but clearly they do not.

Reply:
Yes, the flexible schema is expecting everything in one yaml, and looking by work model to find anything linked to that model. The simple schema is loading each yaml that is defined.

Reply:
I think it’s reasonable to expect `#index_rules_for` to camelize its argument in the M3 loader, if the expectation is that M3 class names are camelized.